ci-operator/step-registry/nutanix/upgrade/procedure/nutanix-upgrade-procedure-ref.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "nutanix/upgrade/procedure/nutanix-upgrade-procedure-ref.yaml",
```

Comment:
how about renaming `procedure` to `creds`, or `pre-config`

Reply:
I'm really bad at naming things, `procedure` was suggested by Zhaohua https://github.com/openshift/release/pull/41579#issuecomment-1648972947, I'd like to get your opinions of the final name. `procedure`, `creds`, or `pre-config`? @jianlinliu @sunzhaohua2

Reply:
I prefer pre-config, @sunzhaohua2 wdyt?

Reply:
ok to me

Reply:
Thank you @jianlinliu @sunzhaohua2 Let me update the name.